From 6dc4a3ea9eb3e410e93b0c61c08bd648c8954556 Mon Sep 17 00:00:00 2001
From: Andy Green <andy@openmoko.com>
Date: Fri, 25 Jul 2008 23:06:06 +0100
Subject: [PATCH] tracking-2.6.25-upstream-s3c2410_gpio_irq2pin.patch

Holger got beaten to it on mainline, they provide a
s3c2410_gpio_irq2pin() instead of the name he used.

Signed-off-by: Andy Green <andy@openmoko.com>
---
 include/asm-arm/arch-s3c2410/gpio.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/asm-arm/arch-s3c2410/gpio.h b/include/asm-arm/arch-s3c2410/gpio.h
index 1b96c4e..520c24b 100644
--- a/include/asm-arm/arch-s3c2410/gpio.h
+++ b/include/asm-arm/arch-s3c2410/gpio.h
@@ -61,7 +61,7 @@ static inline int gpio_direction_output(unsigned gpio, int value)
 #define gpio_to_irq(gpio)		s3c2400_gpio_getirq(gpio)
 #else
 #define gpio_to_irq(gpio)		s3c2410_gpio_getirq(gpio)
-#define irq_to_gpio(irq)		s3c2410_irq_to_gpio(irq)
+#define irq_to_gpio(irq)		s3c2410_gpio_irq2pin(irq)
 #endif
 #endif
 #include <asm-generic/gpio.h>
-- 
1.5.6.3

